packages/admin/cms-admin/src/pages/pageTreeSelect/PageTreeSelectDialog.sc.tsx

```diff
@@ -27,8 +27,8 @@ export const PageVisibilityCell = styled(PageTreeCell)`
 
 export const RowClickContainer = styled("div")`
     position: absolute;
-    z-index: 10;
-    left: 0;
+    z-index: 20;
+    left: 40px;
```

Comment:
Fixes a bug where the click on the row only worked on the edges because it was overlayed